code/game/objects/items/nanoforgedkatana.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/nanoforgedkatana
+	name = "Nanoforged Katana"
+	desc = "Glorious space nippon steel, folded a million times, producing the finest blade known to mankind. \
+			This blade seems to transmit knowledge of how to use its powers directly to its owner."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "nanoforgedkatana"
+	item_state = "nanoforgedkatana"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	w_class = WEIGHT_CLASS_BULKY
+	force = 30
+	throwforce = 70
```

Comment:
why the hell is its throwforce so high?

Reply:
Mostly a meme, I wanted to see if it'd be a good idea with someone more aware of balance
The main idea is : "Why would you throw your sword that costs 18 tc at an enemy with 100% chance to embed, meaning they can easily get it back without extra damage, or that you'd have to stop to commit surgery on them yourself once the battle is over?"

I was also maybe planning to add an extra sprite overlay with the sword poking out of the chest, if that ended up being seen as a good funny idea